### ISSUE TITLE
New uploads only visible if browser cache is cleared

### ISSUE THREAD

**Issue by hed101** [filegator/filegator#259]
If uploading new files after a user has already visited the page, this user only sees the new files uploaded if he/she clears the browser cache. Howe can this be avoided?

**Comment by alcalbg** [comment]
Hi,

This sounds like some aggressive caching on the server-side. Check your web server's settings or try with another one.

Anyone else with this problem?

**Comment by hed101** [comment]
Hi there, it's not about server-side caching, it's about browser caching. Tested on Chrome and FireFox.

**Comment by alcalbg** [comment]
Yes, I know, but the server can send a response instructing your browser (with headers) to cache this response and never check again.

Anyway, very strange issue. 

Can you share some more info, request/response logs from the browser, with headers included. Exact PHP version with loaded modules, phpinfo page etc.
